tests/e2e/actions.go

```diff
@@ -1826,6 +1826,73 @@ func (tr TestRun) invokeDoublesignSlash(
 	}
 }
 
+// Cause light client attack evidence for a certain validator to appear on the given chain.
+// The evidence will look like the validator equivocated to a light client.
```

Comment:
nit: maybe it makes sense to also include in the comment a pointer on what exactly is this attack about (e.g., to [CometBFT docs](https://docs.cometbft.com/main/spec/light-client/accountability/#the_misbehavior_of_faulty_validators)).

Reply:
Included, thanks for the suggestion